index.html

```diff
@@ -9,36 +9,214 @@
+          <h4><i class="fas fa-circle"></i> 2015</h4>
+        </div>
+
+        <article>
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+        </article>
+
+        <ul class="languages">
+          <li><a href="#">html</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">javaScript</a></li>
+        </ul>
+
+
+        <button class="see">See Project</button>
+
+
+      </li>
+
+
+      <li class="work three">
+        <img src="./snapshot-portfolio3.png" alt="project-image3">
+         
+         <h3>TONIC</h3>
+
+        <div class="sub-title">
+          <h4>Canopy</h4>
+          <h4><i class="fas fa-circle"></i> Back End Dev</h4>
+          <h4><i class="fas fa-circle"></i> 2015</h4>
+        </div>
+
+        <article>
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+        </article>
+
+        <ul class="languages">
+          <li><a href="#">html</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">javaScript</a></li>
+        </ul>
+
+
+        <button class="see">See Project</button>
+
+      </li>
+
+      <li class="work four">
+        <img src="./snapshot-portfolio4.png" alt="project-image4">
+         
+         <h3>MULTI-POST <br>STORIES</h3>
+
+        <div class="sub-title">
+          <h4>Canopy</h4>
+          <h4><i class="fas fa-circle"></i> Back End Dev</h4>
+          <h4><i class="fas fa-circle"></i> 2015</h4>
+        </div>
+
+        <article>
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+        </article>
+
+        <ul class="languages">
+          <li><a href="#">html</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">javaScript</a></li>
+        </ul>
+
+         <button class="see">See Project</button>
+        
+        </li>
+
+      <li class="about">
+        <h2>About <br>Myself</h2>
+        <p class="about1">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        
+        <div class="contacts"> 
+          <a href="#" class="lets-connect">LET'S CONNECT</a>
+        </div>
+
+        
+        <div class="social-media">
+          <ul class="sm">
+            <li><a href="#" aria-label="twitter"><i class="fab fa-twitter"></i></a></li> 
+            <li><a href="#" aria-label="linkedin"><i class="fab fa-linkedin-in"></i></a></li>
+            <li><a href="#" aria-label="medium"><i class="fab fa-medium-m"></i></a></li>
+            <li><a href="#" aria-label="gtihub"><i class="fab fa-github"></i></a></li> 
+          </ul>
+        </div>
+      
+        <button class="see">Get My Resume</button>
```

Comment:
- [ ] The button should follow the button styling: Active, Hover and Pressed (see design below) and kindly fix that.
![buttonsat](https://user-images.githubusercontent.com/65852621/133895354-5c6dcb0c-c735-42ba-8e61-d88ba93f479f.png)